### PR TITLE
[addons] CAddonDll ctor has to set m_binaryAddonBase. Fixes PVR addon update

### PR DESCRIPTION
**Pull request by ksooo**
Fixes PVR update addon not working. Installed without errors, but old version shown in GUI afterwards and still marked as to update until next Kodi restart.

@AlwinEsch please review

### CODE REVIEW

**Review by AlwinEsch — CHANGES_REQUESTED** [review]
Should work and apart from the two comments it's OK.

I have to look soon to reduce the CAddon / CAddonDll usage to a minimum and to go with CAddonInfo where running addons are not directly associated.

File: xbmc/addons/binary-addons/BinaryAddonManager.cpp [hunk 1]
```diff
@@ -62,6 +62,14 @@ void CBinaryAddonManager::ReleaseAddonBase(const BinaryAddonBasePtr& addonBase,
   m_runningAddons.erase(addon);
 }
 
+BinaryAddonBasePtr CBinaryAddonManager::GetRunningAddonBase(const std::string& addonId) const
+{
+  const auto& addonInstances = m_runningAddons.find(addonId);
```
[the remark this answers is not in the document]
Comment (ksooo) [reply xbmc/addons/binary-addons/BinaryAddonManager.cpp:69]: Fixed.

File: xbmc/addons/binary-addons/BinaryAddonManager.h [hunk 2]
```diff
@@ -61,6 +61,14 @@ namespace ADDON
      */
     void ReleaseAddonBase(const BinaryAddonBasePtr& addonBase, IAddonInstanceHandler* handler);
 
+    /*!
+     * @brief Get running addon instance handle base.
+     *
+     * @param[in] addonId related addon id
+     *
```
[the remark this answers is not in the document]
Comment (ksooo) [reply xbmc/addons/binary-addons/BinaryAddonManager.h:69]: Fixed.

**Review by AlwinEsch — APPROVED** [review]
Is good for me :+1:

**Comment by ksooo** [comment]
jenkins build this please

**Pull request merged**